lib/index.js

```diff
@@ -29,12 +29,41 @@ function defaultHandler(level, error) {
 	const message = JSON.stringify({
 		_level: level,
 		_timestamp: new Date().toISOString(),
-		_commit_hash: config.gitCommit,
+		_commit_hash: config.commitHash,
 		_service: config.service,
 		...formattedError,
 	});
 
-	process.stderr.write(message + '\n');
+	write(level, `${message}\n`, error);
+}
+
+function write(level, message, error) {
+	if (global.window && global.window.console) {
+		const logger = (() => {
+			const consoleLevelMap = {
+				debug: window.console.debug,
+				info: window.console.info,
+				warn: window.console.warn,
+				warning: window.console.warn,
+				error: window.console.error,
+				fatal: window.console.error,
+			};
+
+			return consoleLevelMap[level];
+		})();
+
+		logger(message);
+
+		return;
+	}
+
+	if (process && process.stderr && process.stderr.write) {
+		process.stderr.write(`${message}\n`);
+
+		return;
+	}
+
+	throw error;
```

Comment:
New behaviour I think?

Reply:
It is, however I don't think it's actually possible to hit this (is there a JS env that has neither `process.strerr.write`, nor `window.console.log`? I just wanted to put something there as a fallback if it ever happens rather than it just get lost in the void.

Reply:
Good point 😂